spec/models/conference_preference_spec.rb

```diff
@@ -83,25 +82,29 @@
   end
 
   describe '#terms accepted?' do
```

Comment:
Should the method name have an underscore? #terms_accepted?

Reply:
Yep. I did not look into the details of specs for this PR 😉 

But I can fix it on the way.